### DISCUSSION TITLE
ADD Demo Button

### DISCUSSION THREAD

**Discussion by JuanPabloDiaz** [priyankarpal/projectshut#2415] [Ideas]
I believe that a **_demo_** button will be better than the **_more_** button. And if the user wants to know more about the developer, they could click on the profile picture which could be a new button. Just an idea

I really like the project and the documentation make this project beginner friendly! 

Thanks for the hard work!

**Comment by priyankarpal** [comment]
I think the `more` button represents there are more projects.

**Reply by JuanPabloDiaz** [reply]
Yeah, I got that.

I was just saying that it will be a cool feature to have a new button that will take you directly to the project.

So the card will have buttons for:
- repo
- more
- demo

**Reply by priyankarpal** [reply]
we are only collecting only github repo links, also few projects don't have a demo
